### ISSUE TITLE
[BUG] low contrast text

### ISSUE THREAD

**Issue by vid** [anthropics/claude-code#77808]
### Preflight Checklist

- [x] I have searched [existing issues](https://github.com/anthropics/claude-code/issues?q=is%3Aissue%20state%3Aopen%20label%3Abug) and this hasn't been reported yet
- [x] This is a single bug report (please file separate reports for different bugs)
- [x] I am using the latest version of Claude Code

### What's Wrong?

In vscode extension, code snippet comments render as light green on green and are difficult to read under normal conditions. It does not honour vscode themes so always appears this way.

<img width="1189" height="119" alt="Image" src="https://github.com/user-attachments/assets/3673e620-22c7-4bb3-bd78-16cde0b5f2be" />

see also #48636, #47809

### What Should Happen?

The default colour scheme should be easily readable, and the extension should use theme colours.

### Error Messages/Logs

```shell

```

### Steps to Reproduce

Cause it to generate code with comments.

### Claude Model

Not sure / Multiple models

### Is this a regression?

No, this never worked

### Claude Code Version

2.1.210 

### Platform

Anthropic API

### Operating System

Ubuntu/Debian Linux

### Terminal/Shell

VS Code integrated terminal
